client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-view/components/motion-extension-field/motion-extension-field.component.html

```diff
@@ -16,11 +16,11 @@ <h4>{{ title }}</h4>
     >
         @if (canBeEdited) {
             <mat-basic-chip class="pointer" disableRipple [matMenuTriggerFor]="triggerMenu" [ngClass]="classes">
-                {{ chipValue || '–' | translate }}
+                <span class="wrap-chip">{{ chipValue || '–' | translate }}</span>
             </mat-basic-chip>
         } @else {
-            <mat-basic-chip disableRipple [ngClass]="classes">
-                {{ chipValue | translate }}
+            <mat-basic-chip disableRipple style="text-wrap: wrap" [ngClass]="classes">
```

Comment:
```suggestion
            <mat-basic-chip disableRipple [ngClass]="classes">
```

I tested a bit here and this style does nothing as far as I can see